src/99-appendices/01-contributors.md

```diff
@@ -1,36 +1,36 @@
-| ⚠️    | Tests                                                                             |
-| ✅     | Tutorials                                                                         |
-| 📢    | Talks                                                                             |
-| 📓    | User testing (of new features, tools, etc.)                                       |
-| 📹    | Videos                                                                            |
+| **Emoji** | **Represents**                                                                    |
+| --------- | --------------------------------------------------------------------------------- |
+| 💬        | Answering Questions (on the mailing list, NeuroStars, GitHub, or in person, etc.) |
+| 🐛        | Bug reports                                                                       |
+| 📝        | Blogposts                                                                         |
+| 💻        | Code                                                                              |
+| 🖋        | Content (separate from Blogposts, for example Website news)                       |
+| 📖        | Documentation and specification                                                   |
+| 🔣        | Data (example datasets)                                                           |
+| 🎨        | Design                                                                            |
+| 💡        | Examples (for example datasets, use `Data`)                                       |
+| 📋        | Event Organizers                                                                  |
+| 💵        | Financial Support                                                                 |
+| 🔍        | Funding/Grant Finders                                                             |
+| 🤔        | Ideas & Planning                                                                  |
+| 🚇        | Infrastructure (hosting, build-tools, etc.)                                       |
+| 🚧        | Maintenance of the BIDS standard                                                  |
+| 🧑‍🏫     | Mentoring new contributors                                                        |
+| 🔌        | Plugin/utility libraries                                                          |
+| 📆        | Project management                                                                |
+| 👀        | Reviewed Pull Requests                                                            |
+| 🔧        | Tools                                                                             |
+| 🌍        | Translation                                                                       |
+| ⚠️        | Tests                                                                             |
+| ✅         | Tutorials                                                                         |
```

Comment:
Some weirdness with three of these.

Reply:
will fix those now

FYI: using remark to fix stuff in some files creates so many false positives I had to do things the good old way...

Reply:
@effigies @Remi-Gau this rings a bell --> it might be that these are odd due to the "length" of the emojis. I remember that was screwing something up.

As long as it renders nicely, I suggest to leave this untouched.

Reply:
Ah. Sorry for the noise.

Reply:
oh phewy 
because running remark on this was just reverting the manual change that the CI was giving a green check for... :shrug:

Reply:
will revert that last one...